lizard_connector/__init__.py

```diff
@@ -1 +1,4 @@
 # package
+import lizard_connector.jsdatetime
```

Comment:
Omdat het weinig totale package weinig code is en je zo het package importeert? Zoals: http://mikegrouchy.com/blog/2012/05/be-pythonic-__init__py.html . Of moet ik met __all__ gaan werken, zoals in dat artikel wordt beschreven? Daarnaast zijn connector, query en jdatetime logische gescheiden onderdelen (vooral aangezien dat query deel flink uitgebreid kan worden).